npm/ng-packs/packages/oauth/src/lib/utils/auth-utils.ts

```diff
@@ -9,8 +9,7 @@ import {
   AbpLocalStorageService,
 } from '@abp/ng.core';
 
-const cookieKey = 'rememberMe';
-const storageKey = 'passwordFlow';
+const remember_me = 'remember_me';
```

Comment:
Invalid variable defination, it must be camelCase or UPPER_SNAKE_CASE
Value up to you